packages/react-instantsearch-core/src/connectors/connectRefinementList.js

```diff
@@ -1,3 +1,4 @@
+import { isEqual } from 'lodash';
```

Comment:
We import like this normally to avoid having the whole bundle: `import isEqual from 'lodash/isEqual'` right?